config-templates/redhat/odf-local/changelog.json

```diff
@@ -26,20 +26,20 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
-                "revision": "7",
-                "release-date": "30 October 2023",
-                "cves": "CVE-2023-44487",
-                "golang": "1.20.7",
-                "ubi": "8.8-1072.1697626218",
+                "revision": "8",
+                "release-date": "27 November 2023",
+                "cves": "CVE-2007-4559,CVE-2023-4641,CVE-2023-22745",
```

Comment:
can u add space after commas in CVE list at all places?
`"CVE-2007-4559,CVE-2023-4641,CVE-2023-22745"` ---->
`"CVE-2007-4559, CVE-2023-4641, CVE-2023-22745"`